src/themes/cerulean/collections/menu.overrides

```diff
@@ -1,7 +1,79 @@
-.ui.secondary.menu .item {
-  color: @secondaryItemColor;
+#gradient {
+    .vertical-three-colors(@start-color: #00b3ee; @mid-color: #7a43b6; @color-stop: 50%; @end-color: #c3325f) {
+        background-image: -webkit-linear-gradient(@start-color, @mid-color @color-stop, @end-color);
+        background-image: -o-linear-gradient(@start-color, @mid-color @color-stop, @end-color);
+        background-image: linear-gradient(@start-color, @mid-color @color-stop, @end-color);
+        background-repeat: no-repeat;
+        filter: e(%("progid:DXImageTransform.Microsoft.gradient(startColorstr='%d', endColorstr='%d', GradientType=0)",argb(@start-color),argb(@end-color))); // IE9 and down, gets no color-stop at all for proper fallback
+    }
 }
 
-.ui.tabular.menu .item {
-  color: @secondaryItemColor;
+.btn-shadow(@color) {
```

Comment:
Because I do not understand this when I process the Cerulean theme, so I write the style directly into the `*.overrides` and `*.variables`, to make it work right. After 2-3 themes, I find this LESS common feature and understand what it does, but at last I forget to modify the Cerulean theme in this way. Sorry for this.